packages/docs/src/page-configs/ui-elements/dropdown/specs.md

```diff
@@ -0,0 +1,44 @@
+### Map
+![Example how to get `content.left`](https://user-images.githubusercontent.com/23530004/181629414-5e40211a-7a3c-4d3c-8e78-8808b956d98b.png)
```

Comment:
I didn't find a solution to import local assets. Any suggestions?

Reply:
We'll need a markdownit plugin, that will handle relative paths. Let's have it on hold before massive docs migration to nuxt.